modules/nf-core/jvarkit/sam2tsv/tests/nextflow.config

```diff
@@ -0,0 +1,7 @@
+process {
+
+    withName: JVARKIT_SAM2TSV {
+        ext.args1 =""
+    }
+
```

Comment:
Seems like you can remove this config file, ext.args1 is not referenced in the JVARKIT_SAM2TSV process.